src/redux/reducers/ws/execution.js

```diff
@@ -119,16 +115,23 @@ function reducer(state = getInitialState(), action = {}) {
     }
 
     case types.SET_LIVE_EXECUTION_TRADES: {
-      const { positionData } = payload
+      const { positionData, strategyMapKey, isOpened } = payload
       const { id } = positionData
-      const currentState = state.results?.positions || {}
+      const positions = state.results[strategyMapKey]?.strategy[
+        isOpened ? 'openPositions' : 'closedPositions'
+      ]
+
+      const currentPosition = { ...positions[id] = {}, ...positionData }
```

Comment:
= {} wont work as expected, better to use ...(positions?.[id] || {})

<img width="473" alt="Screenshot 2022-07-12 at 4 43 50 PM" src="https://user-images.githubusercontent.com/29878604/178477412-951d66fc-6378-4ab8-871c-d2c033df9ea4.png">